src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -52,8 +52,9 @@ public GitHubPackage(IServiceProvider serviceProvider)
             await base.InitializeAsync(cancellationToken, progress);
             await EnsurePackageLoaded(new Guid(ServiceProviderPackage.ServiceProviderPackageId));
 
-            //var usageTracker = await GetServiceAsync(typeof(IUsageTracker)) as IUsageTracker;
-            //usageTracker.IncrementLaunchCount();
+            // Activate the usage tracker by forcing an instance to be created.
+            await GetServiceAsync(typeof(IUsageTracker));
```

Comment:
We probably don't have to wait for it to be done, could we just fire and forget like we do with the menus?